libcxx/include/__memory/is_sufficiently_aligned.h

```diff
@@ -28,10 +27,11 @@ _LIBCPP_HIDE_FROM_ABI constexpr bool is_sufficiently_aligned(_Tp* __ptr) {
 #  ifdef _LIBCPP_COMPILER_CLANG_BASED
   return __builtin_is_aligned(__ptr, _Alignment);
 #  else
-  if constexpr (is_constant_evaluated())
+  if consteval {
     return __builtin_constant_p(__builtin_assume_aligned(__ptr, _Alignment) != nullptr);
```

Comment:
I would prefer simply not supporting constexpr with GCC here.
It seems better to fail loudly than to give the wrong result.

Reply:
The goal was to determine whether or not it is implementable, and potentially opening a LWG issue before C++26 is shipped.